x-pack/legacy/plugins/security/server/routes/api/v1/authenticate.js

```diff
@@ -8,14 +8,13 @@ import Boom from 'boom';
 import Joi from 'joi';
 import { schema } from '@kbn/config-schema';
 import { canRedirectRequest, wrapError, OIDCAuthenticationFlow } from '../../../../../../../plugins/security/server';
-import { KibanaRequest } from '../../../../../../../../src/core/server';
-import { createCSPRuleString } from '../../../../../../../../src/legacy/server/csp';
+import { KibanaRequest, createCspDirectives } from '../../../../../../../../src/core/server';
 
 export function initAuthenticateApi({ authc: { login, logout }, __legacyCompat: { config } }, server) {
   function prepareCustomResourceResponse(response, contentType) {
     return response
       .header('cache-control', 'private, no-cache, no-store')
-      .header('content-security-policy', createCSPRuleString(server.config().get('csp.rules')))
+      .header('content-security-policy', createCspDirectives(server.config().get('csp.rules')))
       .type(contentType);
```

Comment:
Should we expose the csp rules in the `SharedGlobalConfig` to allow access to plugins during migration